trunk/SUAVE/Methods/Weights/Buildups/eVTOL/empty.py

```diff
@@ -140,12 +140,11 @@ def empty(config,
     #-------------------------------------------------------------------------------    
     MTOW                = config.mass_properties.max_takeoff  
     output.seats        = config.passengers * 15.   * Units.kg
-    output.passengers   = config.passengers * 70.   * Units.kg
+    output.passengers   = config.passengers * 62.   * Units.kg
```

Comment:
I know it makes your results look really good. But we're not designing around children.

https://www.faa.gov/documentlibrary/media/advisory_circular/ac120-27e.pdf

If you want 62 kg adults then make a default to a realistic value and change your own scripts.

Reply:
Oh I will change it back.